frontend/arrangor-flate/app/components/utbetaling/InnsendtAvUtbetalingDetaljer.tsx

```diff
@@ -0,0 +1,18 @@
+import { Definisjonsliste } from "../Definisjonsliste";
+
+export default function InnsendtAvUtbetalingDetaljer() {
+  return (
+    <Definisjonsliste
+      title={""}
+      headingLevel="3"
+      definitions={[
+        {
+          key: "Innsendt",
+          value: "@TODO",
+        },
+        { key: "Innsendt av", value: "@TODO" },
```

Comment:
Hva er det meningen at skal stå her? Vi har diskutert flere ganger om å ta vare på hvilken person som sendte inn, men bestemt oss for å ikke gjøre det per nå, så jeg synes bare fjern den linjen.